spec/index.html

```diff
@@ -248,5 +249,155 @@ <h2>Basic Concepts</h2>
 
     <p>FIXME.</p>
   </section>
+
+  <section class="appendix normative">
+    <h2>IANA Considerations</h2>
+
+    <p>This section has been submitted to the Internet Engineering Steering
+      Group (IESG) for review, approval, and registration with IANA.</p>
+
+    <h3>application/ld+json</h3>
+    <dl>
+      <dt>Type name:</dt>
+      <dd>application</dd>
+      <dt>Subtype name:</dt>
+      <dd>ld+yaml</dd>
+      <dt>Required parameters:</dt>
+      <dd>N/A</dd>
+      <dt>Optional parameters:</dt>
+      <dd>
+        <dl>
+          <dt><code>profile</code></dt>
```

Comment:
Q: this section is very long. To ensure it is aligned between yaml+ld and json+ld we could probably just reference
the parameter from either the ld+json registration or the json-ld specification document which was used for the registration of ld+json. WDYT?

Reply:
Shortened, now.